5-infrastructure/modules/cloud-sql/data-network.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+data "google_compute_network" "vpc" {
```

Comment:
We should do the data reads in base env so we dont have to do it within each module.

Reply:
makes sense will move it out

Reply:
Done in df7ee2b0ae3063ec554e274365595cf4d6f97f3d